index.js

```diff
@@ -14,11 +14,14 @@ async function run() {
       return
     }
 
+    let deployRepo = core.getInput("deploy-repo")
+    if (!deployRepo) deployRepo = github.context.repo.repo
```

Comment:
This can be replaced with the single line:
`let deployRepo = core.getInput("deploy-repo") ?? github.context.repo.repo`

Also nit in the 2nd line: you shouldn't use falsey statements when you're specifically looking for `null | undefined`. If someone were to modify the code such that `deployRepo` was a non-null but still falsey then it would erroneously default to the context repo. Likely that wouldn't be the case here, though

Reply:
I personally also prefer nullish coalescing. However this was done for consistency to match the two lines in the code above.